sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/certificate/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.1.0"
+VERSION = "1.2.0"
```

Comment:
Why you have sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_version.py and sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/certificate/_version.py?

Reply:
IIRC it has always been like this, and I think it's because we have different API specs for `ledger` and `identity` (see [here](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/confidentialledger/data-plane#confidentialledger)). I just incremented both for consistency.

Reply:
@lynshi any thoughts? is it safe to remove the `sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/certificate/_version.py` version file?

Reply:
I think it is safe; you can build the package to be sure. This is all one module anyway so the _version for certificate is not doing anything.

Reply:
The API spec version is a different parameter independent from the Python package version.

Reply:
Removed the `_version.py` and updated references.